packages/frontend/src/components/staking/components/Validator.js

```diff
@@ -78,7 +78,7 @@ export default function Validator({
     
     const NEARAsTokenWithMetadata = useSelector(selectNEARAsTokenWithMetadata);
 
-    const contractMetadataByContractId = useSelector(selectAllContractMetadata);
+    const contractMetadataByContractId = useSelector(selectContractsMetadata);
```

Comment:
Off topic question: why do we use a `token_id` as a key in a map that's keyed by contract name? Are contract names and token IDs the same thing for fungible tokens?

Reply:
which line of code do you mean?

Reply:
https://github.com/near/near-wallet/blob/WAL-216-refactor-tokens-reducer-into-two-reducers/packages/frontend/src/components/staking/components/Validator.js#L35

Reply:
I see. I'm not sure, it seems like `token_id` is a part of what`get_farms` function is returning -> `StakingFarmContracts.getFarms` - I didn't participate in these changes.